app/models/miq_action.rb

```diff
@@ -799,7 +806,7 @@ def action_cancel_task(action, rec, inputs)
     end
 
     MiqPolicy.logger.info("MIQ(action_cancel_task): Now executing Cancel of task [#{source_event.event_type}] on VM [#{source_event.vm_name}]")
-    ems = ExtManagementSystem.find_by(:id => source_event.ems_id)
+    ems = ExtManagementSystem.find_by_id(source_event.ems_id)
```

Comment:
Why are you changing this?  There have been a number of PRs merged to use the other form: https://github.com/ManageIQ/manageiq/pulls?utf8=%E2%9C%93&q=is%3Apr%20author%3Achessbyte%20is%3Aclosed%20find_by